app/src/main/java/org/glucosio/android/db/DatabaseHandler.java

```diff
@@ -931,45 +924,42 @@ public void deleteCholesterolReading(CholesterolReading reading) {
         return idArray;
     }
 
-    public ArrayList<Integer> getHDLCholesterolReadingAsArray() {
+    public List<Double> getHDLCholesterolReadingAsArray() {
```

Comment:
The method name does not reflect the return type.

Reply:
Also renamed